player.h

```diff
@@ -0,0 +1,22 @@
+#ifndef PLAYER_H
+#define PLAYER_H
+#pragma once
+
+#include "ui_mainwindow.h"
+#include <QMainWindow>
+
+enum Choices { ROCK, PAPER, SCISSORS };
+
+class Player
+{
+public:
+    Player();
+    void intitialize(Ui::MainWindow *ui);
+    void display_choice(Choices player_choice);
+    Choices generate_choice();
+    Choices choice;
```

Comment:
Make it private

Reply:
ok me will do that

Reply:
this'll mean we gotta use some things like a Vtable to access the private method, out side of the class, right?

Reply:
On line 17, variable choice should be private, if you want to access it, use set and get method

Reply:
OK got it

Reply:
done